app/views/admin/theft_alerts/edit.html.haml

```diff
@@ -6,7 +6,18 @@
     url: admin_theft_alert_path(@theft_alert),
     method: :patch,
     html: { class: "m-0" } do |f|
-    = hidden_field_tag :state_transition, params.fetch(:state_transition, "update_details")
+    .form-group
+      = f.label :status, class: "form-label"
+      = f.select :status, options_for_select(TheftAlert.statuses.keys), class: "form-control"
+    .form-group
+      = f.label :theft_alert_plan_id, class: "form-label"
+      = f.select :theft_alert_plan_id, options_for_select(TheftAlertPlan.active.map{|e| [e.title_string, e.id]}), class: "form-control"
+    .form-group
+      = f.label :begin_at, class: "form-label"
```

Comment:
A few requests:

1. Can you automatically set the `begin_at` when the alert is switched to begin, and set the `end_at` based on that?
2. Can you hide the times and only show them after the alert is enabled?
3. Can you use `datetime_local_field`, include a timezone hidden field, and use `TimeParser` to set the time?

Reply:
Updated in  e476085 and 0ecd791